tests/Stripe/Util/DefaultLoggerTest.php

```diff
@@ -1,28 +1,25 @@
 <?php
 
-// Test in a slightly different namespace than usual. See comment on
-// `error_log` below.
 namespace Stripe\Util;
 
 class DefaultLoggerTest extends \Stripe\TestCase
 {
     public function testDefaultLogger()
     {
-        $logger = new DefaultLogger();
-        $logger->error("message");
+        // DefaultLogger uses PHP's `error_log` function. In order to capture
+        // the output, we need to temporarily redirect it to a temporary file.
```

Comment:
I had to completely rewrite this test, as previously it was using a hack where we manually defined an `error_log()` function in the `\Stripe\Util` namespace and relied on the fact that `DefaultLogger` called `error_log()` in its non-qualified form.

The new test actually captures the output of PHP's `\error_log()` function.